docs/components/icon.md

```diff
@@ -1,6 +1,6 @@
 # Icon
 
-**KIcon** - Icon component for displaying SVG symbols.
+**KIcon** - Icon component for displaying SVG symbols. Currently Size, color and viewbox vary between icons, so the implementor must be mindful when using. 
```

Comment:
Did you maybe want to mention how to update/view the docs to verify that the icon is rendering correctly? Or is that too obvious?